### PR TITLE
Updating `kdma_descriptions.yaml` 

### PR DESCRIPTION
**Pull request by jadie1**
- Changed the comparative regression prompt to only include the structured character information listed in `relevant_structured_character_info` in `kdma_descriptions.yaml` rather than including all unique character information
- Improved the QoL `description` and `score_examples` in `kdma_descriptions.yaml`

Note: Still benchmarking to see if `injuries` should be in the `relevant_structured_character_info` for Adept kdmas.

### CODE REVIEW

File: align_system/configs/adm/outlines_regression_aligned_comparative/incontext_phase1.yaml [hunk 1]
```diff
@@ -9,8 +9,9 @@ inference_kwargs:
     leave_one_out_strategy: null
     normalization: null
     sort_actions: true
+    most_similar_first: false
     datasets:
       QualityOfLife: /data/shared/samba/phase1_icl/qol_all_20241106.json
       PerceivedQuantityOfLivesSaved: /data/shared/samba/phase1_icl/vol_all_20241106.json
-      Moral judgement: /data/shared/samba/phase1_icl/moral_judgement_20241106.json
+      Moral judgement: /data/shared/samba/phase1_icl/moral_judgement_20241112.json
```
Comment (eveenhuis) [inline align_system/configs/adm/outlines_regression_aligned_comparative/incontext_phase1.yaml:16]: Did you regenerate this?

Comment (jadie1) [reply align_system/configs/adm/outlines_regression_aligned_comparative/incontext_phase1.yaml:16]: Yes with the changes from the train files ST made today

Comment (eveenhuis) [reply align_system/configs/adm/outlines_regression_aligned_comparative/incontext_phase1.yaml:16]: Moral judgment is ADEPT though?

Comment (jadie1) [reply align_system/configs/adm/outlines_regression_aligned_comparative/incontext_phase1.yaml]: Okay that's been corrected

**Review by eveenhuis — APPROVED** [review]

**Pull request merged**